src/ebay-fake-menu-button/__tests__/index.spec.tsx

```diff
@@ -23,11 +23,24 @@ describe('<EbayFakeMenuButton>', () => {
             )
             fireEvent.click(wrapper.getByRole('button'))
 
+            expect(spy).toBeCalled()
+        })
+        it('should fire onCollapse event', () => {
+            const wrapper = render(
+                <EbayFakeMenuButton onCollapse={spy}>
+                    <EbayFakeMenuButton />
+                </EbayFakeMenuButton>
+            )
+
+            const button = wrapper.container.querySelector('button')
```

Comment:
Prefer to use RTL `getByRole()`. We already use it on line 51

```suggestion
            const button = wrapper.getByRole('button')
```